workflow-manager/main.go

```diff
@@ -0,0 +1,126 @@
+// workflow-manager looks for batches to be processed from an input bucket,
+// and spins up facilitator jobs to process those batches.
+//
+// Right now workflow-manager is just a stub that demonstrates reading from a
+// GCS bucket and talking to the Kubernetes API.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/rest"
+
+	"cloud.google.com/go/storage"
```

Comment:
Just to make sure you know: for Narnia the ingestion buckets will be in S3, so I think you'll need to use [that SDK](https://docs.aws.amazon.com/sdk-for-go/api/service/s3/) and a client setup with [stscreds](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/stscreds/) so that it can assume the appropriate AWS role (hence the `AWS_ROLE_ARN` and `AWS_ACCOUNT_ID` environment variables wired into the cron job).

Reply:
Yep, I'm aware I need to add S3 support; I just figured since we know we'll need GS anyhow, and I'm learning the whole GCS stack, I'd start with that branch of things.